src/Faker/Provider/zh_CN/Text.php

```diff
@@ -0,0 +1,176 @@
+<?php
+
+namespace Faker\Provider\zh_CN;
+
+class Text extends \Faker\Provider\Text
+{
+    protected static $separator = ''; // not used, use preg_split('//u', $text); instead to split by unicode character
+    protected static $separatorLen = 0;
+
+    /**
+     * All punctuation in $baseText: 、 ， ： ； 。 ？ ！ —— （ ） 《 》 “ ” ‘ ’
+     */
+    protected static $notEndPunct = array('、', '，', '：', '；', '（', '）', '《', '》', '“', '‘');
+    protected static $endPunct = array('。', '？', '！', '—', '》', '”', '’');
+    protected static $notBeginPunct = array('、', '，', '：', '；', '。', '？', '！', '（', '）', '《', '》', '”', '’');
+
+    /**
+     * Title: 从百草园到三味书屋 From Baicao Garden to Sanwei Study (From Hundred-Grass Garden to Three-Taste Study)
+     * Author: 鲁迅 Lu Xun (周树人 Zhou Shuren)
+     * Language: Simplified Chinese
+     * Written on: Sept 18, 1926
+     *
+     * Notice: Some punctuation has been removed. Some paragraph and quotations have be joined together.
+     *
+     * @see https://zh.wikisource.org/zh-hans/%E4%BB%8E%E7%99%BE%E8%8D%89%E5%9B%AD%E5%88%B0%E4%B8%89%E5%91%B3%E4%B9%A6%E5%B1%8B
+     * @var string
+     */
+    protected static $baseText = <<<'EOT'
+我家的后面有一个很大的园，相传叫作百草园。现在是早已并屋子一起卖给朱文公的子孙了，连那最末次的相见也已经隔了七八年，其中似乎确凿只有一些野草；但那时却是我的乐园。
```

Comment:
This text is too short imo, the result won't be random enough. for a Markov chain generator to work, each word / character needs to appear many times in the base text.

Reply:
I have select more text from a collection. I use 4 articles together as base to make it long enough.